spec/std/big/big_decimal_spec.cr

```diff
@@ -169,6 +169,90 @@ describe BigDecimal do
     (-BigDecimal.new(3)).should eq(BigDecimal.new(-3))
   end
 
+  describe "#round" do
+    {% for sign in %w[+ -] %}
+      context "(with {{sign.id}} sign)" do
+        it "returns self if digits are equal or more than significant digits" do
+          BigDecimal.new("{{sign.id}}2.5").round(1)
+            .should eq(BigDecimal.new(BigInt.new({{sign.id}}25), 1))
+        end
+
+        it "rounds :up" do
```

Comment:
```suggestion
        it "rounds up" do
```
The colons can probably be removed now since it's using enums.

Reply:
I prefer it this way, since it denotes passed argument.